codegen/cmd/injection-gen/generators/reconciler_reconciler.go

```diff
@@ -351,10 +357,10 @@ func NewReconciler(ctx {{.contextContext|raw}}, logger *{{.zapSugaredLogger|raw}
 		}
 		if opts.FinalizerName != "" {
 			rec.finalizerName = opts.FinalizerName
-		}
+		}{{if .hasStatus}}
```

Comment:
can we reflow this and {{end}} on individual lines?

Reply:
If we do, it will generate non useful newlines, i prefer to avoid it

Reply:
Who reads generated code? :)
This on the other side — is read :)

Reply:
I think `{{- end}}` will eat the whitespace and everybody wins

Reply:
I tried with 

```
		{{if .hasStatus}}
		if opts.SkipStatusUpdates {
			rec.skipStatusUpdates = true
		}
		{{- end}}
```

But it changed all the generated code...

Reply:
If possible, I would love to avoid changing all the generated code downstream for a new line

Reply:
```
		{{- if .hasStatus}}
		if opts.SkipStatusUpdates {
			rec.skipStatusUpdates = true
		}
		{{- end}}
```

should make everybody happy. Applies to another place a few lines above this as well.

Reply:
This one did it!!!